src/main/java/org/openrewrite/kotlin/internal/KotlinPrinter.java

```diff
@@ -1061,6 +1069,9 @@ private void visitTypeAlias(J.VariableDeclarations typeAlias, PrintOutputCapture
 
         protected void visitStatement(@Nullable JRightPadded<Statement> paddedStat, JRightPadded.Location location, PrintOutputCapture<P> p) {
             if (paddedStat != null) {
+                if (paddedStat.getElement().getMarkers().findFirst(ConstructorDelegation.class).isPresent()) {
+                    return;
+                }
```

Comment:
I am not quite happy with this part. I wonder if we should introduce an additional general purpose `Synthetic` (or `OmitStatement`, `Implied`, or similar) marker, which we additionally could add to the `this()` or `super()` call and then here check for that one instead. What do you think, @traceyyoshima ?